### PR TITLE
ffmpeg-qsv: added max_frame_size for avc enc

### PR DESCRIPTION
**Pull request by yefengxx**
added max_frame_size for ffmpeg-qsv avc enc

Signed-off-by: Xu Yefeng <yefengx.xu@intel.com>

### CODE REVIEW

**Comment by yefengxx** [comment]
It depends on https://github.com/intel/vaapi-fits/pull/395

**Comment by yefengxx** [comment]
@uartie @FocusLuo can you review this for me?

File: lib/ffmpeg/encoderbase.py [hunk 1]
```diff
@@ -239,7 +243,7 @@ def check_bitrate(self):
       # acceptable bitrate within 10% of bitrate
       assert(bitrate_gap <= 0.10)
 
-    elif "vbr" == self.rcmode:
+    elif "vbr" == self.rcmode and self.maxFrameSize is None:
```
Comment (uartie) [inline lib/ffmpeg/encoderbase.py:246]: should be `vars(self).get("maxFrameSize", None)`

Comment (yefengxx) [reply lib/ffmpeg/encoderbase.py:246]: ok

**Comment by yefengxx** [comment]
A new PR is submitted in https://github.com/intel/vaapi-fits/pull/398

**Pull request closed without merging**